### PR TITLE
Telcodocs 496

### PR DESCRIPTION
**Pull request by tshwartz**
Version(s):
Main, 4.10, 4.11

Issue:
(https://issues.redhat.com/browse/TELCODOCS-496)

Link to docs preview:
(http://file.emea.redhat.com/tshwartz/TELCODOCS-496/welcome/oke_about.html)

NOTE:
Automatic preview functionality is currently not available.
  * OpenShift documentation team members (core and aligned) must include a link to a locally generated preview for PRs that update the rendered build in any way.
  * External contributors can request a generated preview from the OpenShift documentation team. --->

Additional information:

### CODE REVIEW

**Comment by zanetworker** [comment]
LGTM

**Comment by tbuskey** [comment]
LGTM

File: modules/images-configuration-blocked-payload.adoc [hunk 1]
```diff
@@ -0,0 +1,70 @@
+//Modules included in the following assemblies
```
Comment (bergerhoffer) [inline modules/images-configuration-blocked-payload.adoc:1]: It looks like this file and the updates to image-configuration.adoc were accidentally pulled into your commit/PR. I'm guessing that you accidentally squashed your commit into the commit ahead of yours (which was Kelly's).

There are a few ways that you can resolve this. If you ping me Monday (I'm on ~815am eastern) I'd be happy to walk you through one of the ways to fix it!

Comment (tshwartz) [reply modules/images-configuration-blocked-payload.adoc]: A few last updates to the diagram - file replaced.

**Comment by bergerhoffer** [comment]
Looks good, merging!

**Pull request merged**